watchMeTests/welcomeToSuitest.test.js

```diff
@@ -23,44 +20,41 @@ describe('Basic functionality', async() => {
 		await snippetHomepageOpened();
 	});
 
-	it('should work correctly', async() => {
+	it('This is a sample test case on a sample app called WatchMe.', async() => {
 		/**
 		 * Now that the application is ready to receive user input, send a key to it.
-		 * Below is the simplest version of the "Press" line that sends a navigational key press to the WatchMe app.
+		 * Below is the simplest version of the "Press" line that sends a navigational key press to the WatchMe app. Click on the "once" below to check out the other versions of the "Press" line.
 		 *
 		 */
-		await suitest.press(VRC.RIGHT);
+		await assert.press(VRC.RIGHT).interval(1000);
 		/**
-		 * Pressing "Right" should have highlighted a first image on the WatchMe homepage.
-		 * Below we will verify that this indeed did happen. There are 2 interesting things about the line below:
+		 * Pressing "Right" should have highlighted the second video on the WatchMe homepage.
+		 * Below we will verify that this indeed did happen. There are 3 interesting things about the line below:
 		 *
-		 * 1. Since we have sent some input to the line we must allow it some time to process it.
-		 * This is why we are waiting until our element gets the highlight.
-		 * Execution times will vary depending on the device. In this case we are allowing (an exaggerated) up to 2 seconds
-		 * for the device to respond.
+		 * 1. Since we have sent some input to the app we must allow it some time to process. This is why we are waiting until our element gets the highlight. Execution times will vary depending on the device. In this case, we are allowing (an exaggerated) up to 2 seconds timeout for the device to respond.
 		 *
-		 * 2. This type of the line is called "Assertion". In this case the subject of this assertion
-		 * is the "view element" - a piece of the app's layout. In Suitest you store your view elements
-		 * in an " Element repository" or you can pass selector to element chain.
+		 * 2. This type of line is called "Assertion". In this case, the subject of this assertion is the "view element" - a piece of the app's layout. In Suitest you store your view elements in the "Element repository" accessible from the black menu bar at the top.
 		 *
+		 * 3. In Suitest the assertion also contains a control directive. That is you can tell Suitest how to proceed based on the result of that assertion. Click on that "continue" at the end of the line below to see all warnings.
```

Comment:
Don't need "Click on that "continue" at the end of the line below to see all warnings."